plugins/destiny.py

```diff
@@ -943,6 +943,46 @@ def triumphs(text,nick,bot):
             output.append(', '.join(missing))
     return ' '.join(output)
 
+@hook.command('news')
+def news(text):
+    feed_base_url = 'https://www.bungie.net/en/Rss/News?category='
+    feed_curr = '&currentpage='
+    feed_items_per = '&itemsPerPage=1'
+
+    feeds = {       #Current RSS Feeds breakouts.
+    'all'        : '',   #ALL
+    'news'       : 'news-destiny',
+    'bungie'     : 'news-bungie',
+    'community'  : 'news-community',
+    'updates'    : 'news-updates'
+    }
+
+    if text.find(' ') == -1:
+        rss = text
+        item_number = 0
+    else:
+        rss, item_number = text.split()
+        item_number = int(item_number)
+
+    if item_number > 100:
+        return "I'll only check the last \x02100\x02 posts."
+
+    for item, key in feeds.items():
+        if rss == item:
+            ask = key
+
+    if item_number == 0: 
+        url = ('https://www.bungie.net/en/rss/News?currentpage=1&itemsPerPage=1')
+    else:
+        url = (feed_base_url + rss + feed_curr + str(item_number) + feed_items_per)
```

Comment:
Should be:
url = (feed_base_url + ask + feed_curr + str(item_number) + feed_items_per)